docs/README.md

```diff
@@ -0,0 +1,256 @@
+# Microsoft Azure SDK for Ruby
+
+Microsoft Azure SDK for Ruby has been deprecated. It will be in maintenance mode for a period of time. This doc instructs how to use Azure REST API as a replacement. By the way, it is highly recommanded you to use GA SDKs
+* [Python](https://github.com/Azure/azure-sdk-for-python)
+* [Java](https://github.com/Azure/azure-sdk-for-java)
+* [JavaScript](https://github.com/Azure/azure-sdk-for-js)
+* [.Net](https://github.com/Azure/azure-sdk-for-net)
+
+You can also find other SDKs in [Azure SDK](https://github.com/Azure/azure-sdk).
+
+# Azure REST API
+
+[Azure REST API Reference](https://docs.microsoft.com/en-us/rest/api/azure/)
+
+
+## Authentication
+
+The first step to using the SDK is authentication and permissions. For people unfamilar with Azure this may be one of
```

Comment:
so we are still relying on ms_rest_azure to use the REST API ? if this is the case, we probably should mention it at the beginning as well